### ISSUE TITLE
Proliferation of augmentation dots on chord notes

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#569]
Something funny happened recently: two augmentation dots can appear after notes in chords.  Mostly when the note is on a staff line (at least when the stem is upwards), but also sometimes on notes in spaces with the stem down:

<img width="871" alt="screen shot 2017-04-28 at 9 29 27 pm" src="https://cloud.githubusercontent.com/assets/3487289/25552890/3a1e52c2-2c5a-11e7-8b94-115c2702210a.png">

MEI test data:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="3.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title />
            </titleStmt>
            <pubStmt />
        </fileDesc>
        <encodingDesc>
            <appInfo>
                <application isodate="2017-04-28T21:30:42" version="0.9.14-dev-18df18e">
                    <name>Verovio</name>
                    <p>Transcoded from Humdrum</p>
                </application>
            </appInfo>
        </encodingDesc>
        <workDesc>
            <work>
                <titleStmt>
                    <title />
                </titleStmt>
            </work>
        </workDesc>
    </meiHead>
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef xml:id="scoredef-0000000028106328">
                        <staffGrp xml:id="staffgrp-0000000051881667">
                            <staffDef xml:id="staffdef-0000000096816587" n="1" clef.shape="G" clef.line="2" lines="5" />
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-0000000468036204">
                        <measure xml:id="measure-L3" n="1" right="end">
                            <staff xml:id="staff-L3F1N1" n="1">
                                <layer xml:id="layer-L3F1N1" n="1">
                                    <chord xml:id="chord-L4F1" dots="1" dur="4">
                                        <note xml:id="note-L4F1S1" oct="4" pname="c">
                                            <accid xml:id="accid-L4F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L4F1S2" oct="4" pname="d">
                                            <accid xml:id="accid-L4F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L5F1" dots="1" dur="4">
                                        <note xml:id="note-L5F1S1" oct="4" pname="c">
                                            <accid xml:id="accid-L5F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L5F1S2" oct="4" pname="e">
                                            <accid xml:id="accid-L5F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L6F1" dots="1" dur="4">
                                        <note xml:id="note-L6F1S1" oct="4" pname="c">
                                            <accid xml:id="accid-L6F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L6F1S2" oct="4" pname="f">
                                            <accid xml:id="accid-L6F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L7F1" dots="1" dur="4">
                                        <note xml:id="note-L7F1S1" oct="4" pname="c">
                                            <accid xml:id="accid-L7F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L7F1S2" oct="4" pname="g">
                                            <accid xml:id="accid-L7F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L8F1" dots="1" dur="4">
                                        <note xml:id="note-L8F1S1" oct="4" pname="c">
                                            <accid xml:id="accid-L8F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L8F1S2" oct="4" pname="a">
                                            <accid xml:id="accid-L8F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L9F1" dots="1" dur="4">
                                        <note xml:id="note-L9F1S1" oct="4" pname="c">
                                            <accid xml:id="accid-L9F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L9F1S2" oct="4" pname="b">
                                            <accid xml:id="accid-L9F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L10F1" dots="1" dur="4">
                                        <note xml:id="note-L10F1S1" oct="4" pname="c">
                                            <accid xml:id="accid-L10F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L10F1S2" oct="5" pname="c">
                                            <accid xml:id="accid-L10F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L11F1" dots="1" dur="4">
                                        <note xml:id="note-L11F1S1" oct="4" pname="c">
                                            <accid xml:id="accid-L11F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L11F1S2" oct="5" pname="d">
                                            <accid xml:id="accid-L11F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L12F1" dots="1" dur="4">
                                        <note xml:id="note-L12F1S1" oct="4" pname="c">
                                            <accid xml:id="accid-L12F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L12F1S2" oct="5" pname="e">
                                            <accid xml:id="accid-L12F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L13F1" dots="1" dur="4">
                                        <note xml:id="note-L13F1S1" oct="4" pname="c">
                                            <accid xml:id="accid-L13F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L13F1S2" oct="5" pname="f">
                                            <accid xml:id="accid-L13F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L14F1" dots="1" dur="4">
                                        <note xml:id="note-L14F1S1" oct="4" pname="c">
                                            <accid xml:id="accid-L14F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L14F1S2" oct="5" pname="g">
                                            <accid xml:id="accid-L14F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L15F1" dots="1" dur="4">
                                        <note xml:id="note-L15F1S1" oct="4" pname="c">
                                            <accid xml:id="accid-L15F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L15F1S2" oct="5" pname="a">
                                            <accid xml:id="accid-L15F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L16F1" dots="1" dur="4">
                                        <note xml:id="note-L16F1S1" oct="4" pname="d">
                                            <accid xml:id="accid-L16F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L16F1S2" oct="5" pname="a">
                                            <accid xml:id="accid-L16F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L17F1" dots="1" dur="4">
                                        <note xml:id="note-L17F1S1" oct="4" pname="e">
                                            <accid xml:id="accid-L17F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L17F1S2" oct="5" pname="a">
                                            <accid xml:id="accid-L17F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L18F1" dots="1" dur="4">
                                        <note xml:id="note-L18F1S1" oct="4" pname="f">
                                            <accid xml:id="accid-L18F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L18F1S2" oct="5" pname="a">
                                            <accid xml:id="accid-L18F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L19F1" dots="1" dur="4">
                                        <note xml:id="note-L19F1S1" oct="4" pname="g">
                                            <accid xml:id="accid-L19F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L19F1S2" oct="5" pname="a">
                                            <accid xml:id="accid-L19F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L20F1" dots="1" dur="4">
                                        <note xml:id="note-L20F1S1" oct="4" pname="a">
                                            <accid xml:id="accid-L20F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L20F1S2" oct="5" pname="a">
                                            <accid xml:id="accid-L20F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L21F1" dots="1" dur="4">
                                        <note xml:id="note-L21F1S1" oct="4" pname="b">
                                            <accid xml:id="accid-L21F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L21F1S2" oct="5" pname="a">
                                            <accid xml:id="accid-L21F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L22F1" dots="1" dur="4">
                                        <note xml:id="note-L22F1S1" oct="5" pname="c">
                                            <accid xml:id="accid-L22F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L22F1S2" oct="5" pname="a">
                                            <accid xml:id="accid-L22F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L23F1" dots="1" dur="4">
                                        <note xml:id="note-L23F1S1" oct="5" pname="d">
                                            <accid xml:id="accid-L23F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L23F1S2" oct="5" pname="a">
                                            <accid xml:id="accid-L23F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L24F1" dots="1" dur="4">
                                        <note xml:id="note-L24F1S1" oct="5" pname="e">
                                            <accid xml:id="accid-L24F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L24F1S2" oct="5" pname="a">
                                            <accid xml:id="accid-L24F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L25F1" dots="1" dur="4">
                                        <note xml:id="note-L25F1S1" oct="5" pname="f">
                                            <accid xml:id="accid-L25F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L25F1S2" oct="5" pname="a">
                                            <accid xml:id="accid-L25F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <chord xml:id="chord-L26F1" dots="1" dur="4">
                                        <note xml:id="note-L26F1S1" oct="5" pname="g">
                                            <accid xml:id="accid-L26F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L26F1S2" oct="5" pname="a">
                                            <accid xml:id="accid-L26F1S2" accid.ges="n" />
                                        </note>
                                    </chord>
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

**Comment by craigsapp** [comment]
Single notes are not affected:

<img width="1268" alt="screen shot 2017-04-28 at 9 35 19 pm" src="https://cloud.githubusercontent.com/assets/3487289/25552906/a430a05c-2c5a-11e7-80e2-0cf10f411273.png">

MEI test data:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="3.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title />
            </titleStmt>
            <pubStmt />
        </fileDesc>
        <encodingDesc>
            <appInfo>
                <application isodate="2017-04-28T21:35:28" version="0.9.14-dev-18df18e">
                    <name>Verovio</name>
                    <p>Transcoded from Humdrum</p>
                </application>
            </appInfo>
        </encodingDesc>
        <workDesc>
            <work>
                <titleStmt>
                    <title />
                </titleStmt>
            </work>
        </workDesc>
    </meiHead>
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef xml:id="scoredef-0000000401634759">
                        <staffGrp xml:id="staffgrp-0000001028471840">
                            <staffDef xml:id="staffdef-0000000430340177" n="1" clef.shape="G" clef.line="2" lines="5" />
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-0000001804473882">
                        <measure xml:id="measure-L3" n="1" right="end">
                            <staff xml:id="staff-L3F1N1" n="1">
                                <layer xml:id="layer-L3F1N1" n="1">
                                    <chord xml:id="chord-L4F1" dots="1" dur="4">
                                        <note xml:id="note-L4F1S1" oct="4" pname="c">
                                            <accid xml:id="accid-L4F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L4F1S2" oct="4" pname="f">
                                            <accid xml:id="accid-L4F1S2" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L4F1S3" oct="4" pname="b">
                                            <accid xml:id="accid-L4F1S3" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L4F1S4" oct="5" pname="e">
                                            <accid xml:id="accid-L4F1S4" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L4F1S5" oct="5" pname="a">
                                            <accid xml:id="accid-L4F1S5" accid.ges="n" />
                                        </note>
                                    </chord>
                                    <note xml:id="note-L5F1" dots="1" dur="4" oct="4" pname="c">
                                        <accid xml:id="accid-L5F1" accid.ges="n" />
                                    </note>
                                    <note xml:id="note-L6F1" dots="1" dur="4" oct="4" pname="d">
                                        <accid xml:id="accid-L6F1" accid.ges="n" />
                                    </note>
                                    <note xml:id="note-L7F1" dots="1" dur="4" oct="4" pname="e">
                                        <accid xml:id="accid-L7F1" accid.ges="n" />
                                    </note>
                                    <note xml:id="note-L8F1" dots="1" dur="4" oct="4" pname="f">
                                        <accid xml:id="accid-L8F1" accid.ges="n" />
                                    </note>
                                    <note xml:id="note-L9F1" dots="1" dur="4" oct="4" pname="g">
                                        <accid xml:id="accid-L9F1" accid.ges="n" />
                                    </note>
                                    <note xml:id="note-L10F1" dots="1" dur="4" oct="4" pname="a">
                                        <accid xml:id="accid-L10F1" accid.ges="n" />
                                    </note>
                                    <note xml:id="note-L11F1" dots="1" dur="4" oct="4" pname="b">
                                        <accid xml:id="accid-L11F1" accid.ges="n" />
                                    </note>
                                    <note xml:id="note-L12F1" dots="1" dur="4" oct="5" pname="c">
                                        <accid xml:id="accid-L12F1" accid.ges="n" />
                                    </note>
                                    <note xml:id="note-L13F1" dots="1" dur="4" oct="5" pname="d">
                                        <accid xml:id="accid-L13F1" accid.ges="n" />
                                    </note>
                                    <note xml:id="note-L14F1" dots="1" dur="4" oct="5" pname="e">
                                        <accid xml:id="accid-L14F1" accid.ges="n" />
                                    </note>
                                    <note xml:id="note-L15F1" dots="1" dur="4" oct="5" pname="f">
                                        <accid xml:id="accid-L15F1" accid.ges="n" />
                                    </note>
                                    <note xml:id="note-L16F1" dots="1" dur="4" oct="5" pname="g">
                                        <accid xml:id="accid-L16F1" accid.ges="n" />
                                    </note>
                                    <note xml:id="note-L17F1" dots="1" dur="4" oct="5" pname="a">
                                        <accid xml:id="accid-L17F1" accid.ges="n" />
                                    </note>
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

**Comment by lpugin** [comment]
Fixed 40d7dc8f1e0728f7dc027e8e9de5875e754a5a97

**Comment by craigsapp** [comment]
Looks good:

<img width="1388" alt="screen shot 2017-04-29 at 4 59 49 am" src="https://cloud.githubusercontent.com/assets/3487289/25555343/ba6d7ee2-2c98-11e7-9d31-f4482dc45f62.png">
